spotifycli/spotifycli.py

```diff
@@ -49,6 +54,26 @@ def main():
         control_volume("-5%")
 
 
+def startShell():
+    while(True):
+        try:
+            command = input('Spotify > ')
+        except EOFError as exception:
+            print("Have a nice day!")
+            exit(0)
+
+        pid = os.fork()
+
+        if pid == 0:  # if executing context is child process
+            os.execlp('spotifycli', 'spotifycli', '--{}'.format(command))
+        elif pid > 0:
+            exit_status = os.waitpid(pid, 0)[1]  # wait for child to exit
```

Comment:
Is `exit_status` variable used anywhere? Is it any implicit value used by Python under the hood? If not, we can remove variable assignment.

Reply:
It did do something at some point. If an invalid command is ran, it shows the regular help menu which doesn't make sense from the context of the shell mode.

I was initially trying to redirect stderr +stdout to a file in /tmp, and then checking if the child exited normally. If it did, it read that file in /tmp, otherwise gave a custom error. The redirect didn't work and I was left confused because it looked like it would've worked.

What are your thoughts on the menu for the shell mode?

Reply:
but is there any purpose of keeping this variable? I mean, if you just call `os.waitpid(pid, 0)[1]` instead of `exit_status = os.waitpid(pid, 0)[1]`, then is there any difference?

I think menu could be nice (depending on how it will be implemented), but in a separate PR.

Reply:
There is no purpose in keeping it now. I'll remove it and address your other comments soon